src/core/issues-notes.js

```diff
@@ -23,6 +23,16 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
+    if (!(Object.keys(ghIssues).length === 0 && ghIssues.constructor === Object)) {
+      $ins = $ins.filter(function (index, item) {
+        var dataNum = item.getAttribute("data-number");
+        if (ghIssues[dataNum] === undefined) {
+          item.remove();
+        }
+        pub("warn", `issue #${dataNum} is a closed issue`);
+        return ghIssues[dataNum] !== undefined;
```

Comment:
How about:

```
if issue is defined {
  return true;
}
warn about the undefined thing;
```

This way we don't need to do `[=|!]== undefined` twice.

Reply:
Ok